gematria/datasets/find_accessed_addrs_exegesis.cc

```diff
@@ -61,6 +61,12 @@ using namespace llvm::exegesis;
 constexpr uint64_t kInitialRegVal = 0x12345600;
 constexpr uint64_t kInitialMemVal = 0x12345600;
 
+// TODO(boomanaiden154): This is currently chosen just to make things simple.
+// Eventually we should probably switch to a different constant per FP
+// FP register size and ensure it is a reasonable enough value to hopefully
+// avoid problems like expensive FP operations due to edge cases.
+constexpr uint64_t kInitialFloatVal = 0x12345600;
```

Comment:
Does this produce a reasonable floating point value? (We might want to do a `bit_cast` from a double just to be safe).

Reply:
Used `llvm::bit_cast` to get a reasonable value (10^6) for floats and doubles. Will need some more thought on how exactly we do this in the future, so left the comment (modified).

This is mostly just aimed at a stop-gap so we can avoid MC verification errors when annotating.